lib/matplotlib/tests/test_axes.py

```diff
@@ -5091,11 +5091,22 @@ def test_broken_barh_empty():
     ax.broken_barh([], (.1, .5))
 
 
-def test_pandas_pcolormesh():
+@pytest.fixture
+def pd(request):
+    '''fixuture to import and configure pandas'''
+
     pd = pytest.importorskip('pandas')
-    from pandas.tseries import converter
-    converter.register()
+    if pd:
+        try:
+            from pandas.plotting import (
+                register_matplotlib_converters as register)
+        except ImportError:
+            from pandas.tseries.converter import register
+        register()
```

Comment:
Should you unregister after the test is run? Is that even possible? Or is it covered by the general reset fixture?

Reply:
For newer versions of pandas we can.  Update coming.